packages/synapse-react-client/src/components/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,80 @@
+import IconSvg from '@/components/IconSvg/IconSvg'
+import { truncateString } from '@/utils/functions/StringUtils'
+import {
+  Breadcrumbs as MuiBreadcrumbs,
+  Link,
+  SxProps,
+  Tooltip,
+  Typography,
+} from '@mui/material'
+import { MouseEventHandler } from 'react'
+import { Link as RouterLink } from 'react-router'
+
+export type BreadcrumbItem = {
+  /* The text to show in the breadcrumb item. Strings > 25 characters will be truncated */
+  text: string
+  /* Whether this item represents the current page. If true, this item will not have a link. Default false */
+  current?: boolean
+  /* Link for the item, if using react-router */
+  to?: string
+  /* Link for the item, if it should be a normal anchor */
+  href?: string
+  /* Event handler fired when the link is clicked */
+  onClick?: MouseEventHandler
+}
+
+type BreadcrumbsProps = {
+  /** The items to show in the breadcrumbs */
+  items: BreadcrumbItem[]
+  /** sx passed to the MUI Breadcrumbs */
+  sx?: SxProps
+  /** If defined, truncate individual item strings to be no longer than this value */
+  maxBreadcrumbLength?: number
+}
+
+function BreadcrumbSeparator() {
+  return <IconSvg icon="chevronRight" />
+}
+
+export function Breadcrumbs(props: BreadcrumbsProps) {
```

Comment:
Refactored existing `EntityPageBreadcrumbs` component to put logic here. Styles extracted. We now reuse this in the new Access Requests pages with different styles.